website/src/components/BlogListItem.tsx

```diff
@@ -22,7 +22,7 @@ export const BlogListItem: FunctionComponent<Props> = ({ blog }) => (
             <p>{blog.description}</p>
         </div>
         <div className="col-sm-4 col-md-3 d-flex align-items-center">
-            {blog.image && <img className="flex-1 w-100" src={blog.image} />}
+            {blog.image && <img className="flex-1 w-100" alt="Resource thumbnail" src={blog.image} />}
```

Comment:
I think we should revise the interface for image to have a `src` and an `alt` so we can be more descriptive about the alt text for SEO vs being generic.

Reply:
Good point. What do you think of the `alt`'s I added here? [87b115a](https://github.com/sourcegraph/about/pull/5271/commits/87b115af9d2e0af8d805b672e71c36a9a6ba5981)

Reply:
Sweet, yeah that works!